src/encoded/schemas/hic_quality_metric.json

```diff
@@ -17,127 +17,458 @@
-            "minimum": 0
+        "pct_1_alignment_unique": {
+            "title": "Percentage of 1 alignment unique",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 100.0
         },
-        "pct_chimeric_ambiguous": {
-            "title": "Percentage of chimeric ambiguous",
+        "pct_1_alignment_duplicates": {
+            "title": "Percentage of 1 alignment duplicates",
             "type": "number",
             "minimum": 0,
             "maximum": 100.0
         },
-        "unmapped_reads": {
-            "title": "Number of unmapped reads",
-            "type": "integer",
-            "minimum": 0
+        "pct_sequenced_1_alignment_unique": {
+            "title": "Percentage of 1 alignment unique",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 100.0
         },
-        "pct_unmapped_reads": {
-            "title": "Percentage of unmapped reads",
+        "pct_sequenced_1_alignment_duplicates": {
+            "title": "Percentage of 1 alignment duplicates",
             "type": "number",
             "minimum": 0,
             "maximum": 100.0
         },
-        "ligation_motif_present": {
-            "title": "Number of reads with ligation motif present",
-            "type": "integer",
-            "minimum": 0
+        "pct_2_alignments": {
+            "title": "Percentage of 2 alignments",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 100.0
         },
-        "pct_ligation_motif_present": {
-            "title": "Percentage of reads with ligation motif present",
+        "pct_2_alignment_unique": {
+            "title": "Percentage of 2 alignment unique",
             "type": "number",
             "minimum": 0,
             "maximum": 100.0
         },
-        "single_alignment": {
-            "title": "Number of single alignments",
-            "type": "integer",
-            "minimum": 0
+        "pct_2_alignment_duplicates": {
+            "title": "Percentage of 2 alignment duplicates",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 100.0
         },
-        "pct_single_alignment": {
-            "title": "Percentage of single alignments",
+        "pct_2_alignments_a_b": {
+            "title": "Percentage of 2 alignments a_b",
             "type": "number",
             "minimum": 0,
             "maximum": 100.0
         },
-        "avg_insert_size": {
-            "title": "Average of the insert size",
+        "pct_2_alignments_a1_a2b_a1b2_b1a2": {
+            "title": "Percentage of 2 alignments a1_a2b_a1b2_b1a2",
             "type": "number",
-            "minimum": 0
+            "minimum": 0,
+            "maximum": 100.0
         },
-        "alignable_normal_and_chimeric_paired": {
-            "title": "Number of alignable paired reads",
-            "type": "integer",
-            "minimum": 0
+        "pct_sequenced_2_alignment_unique": {
```

Comment:
I think we also need to add a property for `pct_sequenced_2_alignment_duplicates`